src/Goerli-DssSpell.t.base.sol

```diff
@@ -2294,6 +2294,28 @@ contract GoerliDssSpellTestBase is DSTest, DSMath {
         vat.move(address(this), address(0x0), vat.dai(address(this)));
     }
 
+    function getMat(bytes32 _ilk) internal returns (uint256 mat) {
+        (, mat) = spotter.ilks(_ilk);
+    }
+
+    function checkIlkLerpOffboarding(bytes32 _ilk, bytes32 _lerp, uint256 _startMat, uint256 _lowMidMat, uint256 _highMidMat, uint256 _endMat) public {
+        vote(address(spell));
+        scheduleWaitAndCast(address(spell));
+        assertTrue(spell.done());
+
+        LerpAbstract lerp = LerpAbstract(lerpFactory.lerps(_lerp));
+
+        hevm.warp(block.timestamp + lerp.duration() / 2);
+        assertEq(getMat(_ilk), _startMat * RAY / 100);
+        lerp.tick();
+        assertGt(getMat(_ilk), _lowMidMat * RAY / 100);
+        assertLt(getMat(_ilk), _highMidMat * RAY / 100);
```

Comment:
There is an `assertEqAprox` in the charter PR you could copy and paste.
Regardless of which function you use I'd suggest making the range tighter.

Reply:
Done in b82e02d.